src/components/input/reactive/Select.tsx

```diff
@@ -27,7 +27,7 @@ const Select = ({
   };
 
   useEffect(() => {
-    if (!props.options[0]) {
+    if (!props.options[0] || !field.value) {
```

Comment:
!field.value isn't the correct solution. If we don't have a value (on create) and we have a required field then we have to assign the first option